### PR TITLE
chore(SearchComponent): codemod for migration of SearchComponent import to Search

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/6887605077
https://monday.monday.com/boards/3532714909/pulses/7096885192

### CODE REVIEW

File: packages/core/docs/vibe-3-changelog.md [hunk 1]
```diff
@@ -62,35 +58,48 @@ codemod: `avatar-component-migration`
...
-- Changed default size to "small" instead of "medium" [codemod - change usages of size medium to small, keep other usages the same]
 - Fixed "medium" size spacings
+- Changed default size to "small" instead of "medium" [codemod - change usages of size medium to small, keep other usages the same]
 
 ### Divider
 
 - `classname` -> `className` [codemod]
 
-### Dropdown
+### Dropdown 🚩
```
[the remark this answers is not in the document]
Comment (YossiSaadi) [reply packages/core/docs/vibe-3-changelog.md:72]: Codemod for Dropdown - change Dropdown.size static prop to Dropdown.sizes
https://monday.monday.com/boards/6841644723/pulses/7109094971

File: packages/core/docs/vibe-3-changelog.md [hunk 2]
```diff
@@ -62,35 +58,48 @@ codemod: `avatar-component-migration`
...
-- Spin in animation removed due to UX decision, including `isAnimationDisabled` prop which is not needed anymore
+- Spin in animation removed due to UX decision
+  - Removed `isAnimationDisabled` prop which is not needed anymore [codemod]
 
 ### EditableHeading
 
 - Complete API Change (TBD)
 
-### Link
+### Link 🚩
```
Comment (YossiSaadi) [inline packages/core/docs/vibe-3-changelog.md:102]: Codemod infra for changing a prop value with static prop namespace
[monday.monday.com/boards/6841644723/pulses/7109091061](https://monday.monday.com/boards/6841644723/pulses/7109091061)

Comment (YossiSaadi) [reply packages/core/docs/vibe-3-changelog.md:102]: Codemod for Link - change Link.target static prop to Link.targets
https://monday.monday.com/boards/6841644723/pulses/7109104112

File: packages/core/docs/vibe-3-changelog.md [hunk 3]
```diff
@@ -143,32 +154,40 @@ codemod: `avatar-component-migration`
...
-- Requires codemod for when people used secondaryDialogPosition={SplitButton.secondaryBlaBla}. [codemod]
-- Once changing DialogPosition to be a const, instead of enum, remove DialogPosition double declaration from SplitButton's declaration [internal]
-- Remove `data-testId` prop from nested elements
+- [!!!] We're now accepting instead of static props, inline string, e.g. "bottom-start". We need to change DialogPosition to be a string in its root declaration (this also requires big codemod changes probably) [codemod]
+  - Requires codemod for when people used secondaryDialogPosition={SplitButton.secondaryBlaBla}. [codemod]
+  - Once changing DialogPosition to be a const, instead of enum, remove DialogPosition double declaration from SplitButton's declaration [internal]
+- Remove `data-testId` prop from nested elements (data-testId used to be applied also to each of the internal elements)
 
-### Steps
+### Steps 🚩
```
Comment (YossiSaadi) [inline packages/core/docs/vibe-3-changelog.md:170]: Codemod infra for changing a prop value
https://monday.monday.com/boards/6841644723/pulses/7109083611

Comment (YossiSaadi) [reply packages/core/docs/vibe-3-changelog.md:170]: Codemod for Steps - replace isOnPrimary boolean flag prop to color prop with value of Steps.colors.PRIMARY
https://monday.monday.com/boards/6841644723/pulses/7109110438

File: packages/core/docs/vibe-3-changelog.md [hunk 4]
```diff
@@ -127,17 +136,19 @@ codemod: `avatar-component-migration`
...
-
-### ModalHeader
-
-- `hideCloseButton` removed due to UX decision, Modals should always have a close button [codemod]
+- Removed `disabledReason`, use `tooltipContent` instead [codemod]
 
 ### Modal
 
-- `hideCloseButton` removed due to UX decision, Modals should always have a close button [codemod]
+- Modal no longer have the ability to have a close button due to UX decision
```
Comment (talkor) [inline packages/core/docs/vibe-3-changelog.md:143]:
```suggestion
- Modal no longer have the ability to remove the close button due to UX decision
```
@YossiSaadi

**Review by talkor — APPROVED** [review]

**Pull request merged**